examples/layers/widgets/spinning_mixed.dart

```diff
@@ -52,10 +52,10 @@ void attachWidgetTreeToRenderTree(RenderProxyBox container) {
                   mainAxisAlignment: MainAxisAlignment.spaceAround,
                   children: <Widget>[
                     ElevatedButton(
-                      child: Row(
+                      child: const Row(
                         children: <Widget>[
-                          Image.network('https://flutter.dev/images/favicon.png'),
-                          const Text('PRESS ME'),
+                          FlutterLogo(),
```

Comment:
This works, and is probably the best solution.  If you wanted a network image in the future, however, there are some images served by the API docs site that you can use.

See  https://flutter.github.io/assets-for-api-docs/assets/widgets/owl-2.jpg or https://flutter.github.io/assets-for-api-docs/assets/widgets/falcon.jpg , for instance.